doc/source/cluster/kubernetes/user-guides/kubectl-dashboard.md

```diff
@@ -0,0 +1,33 @@
+(kubectl-dashboard)=
+
+# Use kubectl dashboard (alpha)
+
+Starting from KubeRay v1.4.0, you can use the open source dashboard UI for KubeRay.
```

Comment:
can you explain what's KubeRay dashboard? Without the explanation, users don't know the difference with Ray dashboard.